build.yaml

```diff
@@ -2578,17 +2578,6 @@ targets:
   - grpc
   - gpr_test_util
   - gpr
-- name: grpc_invalid_channel_args_test
```

Comment:
This test was dealing with channel args that have been moved down to the subchannel stack. In addition, it was redundant, as the functions that now use to read the value of the relevant channel args would already signal problems this test covered.